src/components/Group/AgreementForm.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div class="edit" :class="{ changed: hasChanged }">
-    <form @submit.prevent="save">
+    <form @submit.prevent="checkSave">
```

Comment:
could get a better name instead of `checkSave`...

Reply:
`maybeSave`?